### ISSUE TITLE
Different results when using gprofiler list of genes in each set vs using user defined expression file

### ISSUE THREAD

**Issue by risserlin** [BaderLab/EnrichmentMapApp#96]
Run EM with gprofiler results (one without an expression file(new file format that contains the set of genes from the query with the associated genes listed and one with the expression file) using all the same parameters and the network (number of nodes and edges) are different.

**Comment by risserlin** [comment]
Issue is related to a discrepancy in the gprofiler gene column and the gprofiler gmt file.  There is no way to fix it from our end.  the gprofiler results are sometimes missing genes in the list and sometimes have additional gene not in the original gmt file.  (Might be related to identifier mapping).

I have reported this issue to Juri. 

Not closing this issue as need to check it once hear back from gprofiler. 

result files of comparison:
https://www.dropbox.com/s/kpkx618ynq206e3/oldEM201_vs_newEM210_sizes.xlsx?dl=0
https://www.dropbox.com/s/glw29n1dyf6h49p/GO0031988_moregenesinNewEM210_summary.xlsx?dl=0
https://www.dropbox.com/s/9y6bcdu2gyuuk95/GO0010033_lessgenesinNewE210_summary.xlsx?dl=0
https://www.dropbox.com/s/65d9zc3f7kh0fgb/hsapiens.NAME_2.gmt?dl=0
https://www.dropbox.com/s/hfr1u5yepfxo9ke/gprofiler_results_1000139768493_updated2015.txt?dl=0

**Comment by risserlin** [comment]
Heard back from gprofiler and this boils down to an identifier mapping issue. 

Only way to fix is to click on the warning message in gprofiler and resolve all identifier issues.

Add work around to the gprofiler tutorial and update the results from the gprofiler tutorial based on the new working of  the app.

**Comment by risserlin** [comment]
Response from gprofiler:
Some query identifiers are not present in the output, these are (note
that g:Profiler is case-insensitive):

BOLA2
GPR89B
SERPINA3
SGK3
TXNDC5

If you input these alone and run the g:GOSt web interface, you'll
notice that no results are output, but a dialog appears complaining
that these identifiers are ambiguous and asks the user to resolve
these to ENSG manually. Internally, g:Profiler translates all queries
into the ENSG namespace, but due to ambiguities, this cannot sometimes
be done automatically. Once these are resolved, these identifiers
should appear in the results. Link to the query: http://eid.ee/3tp

One query identifier (C19ORF10) is present in the output, but not the
GMT file. This happens because the GMT files contains only names, i.e.
identifiers in the NAME namespace. If you run g:Convert with this ID
(http://eid.ee/3to), you'll notice that the name corresponding to
C19ORF10 is "MYDGF", which _is_ present in the GMT.

In conclusion, if you wish to avoid such discrepancies, you should
make sure your query list is in the same namespace as the GMT file
you're comparing with (either NAME or ENSG; g:Convert can help with
namespace conversions) _and_ that all ambiguities have been resolved.
The latter is only available in the web tool, if you use the R
package, you should first resolve these ambiguities by hand; g:Convert
can help here as well.

**Comment by risserlin** [comment]
Add disclaimer to tutorial and user guide about ID discrepancies

**Comment by risserlin** [comment]
Added identifier mapping as step in tutorial.
